### ISSUE TITLE
announceTitle provided by futureporn-scout is low quality

### ISSUE THREAD

**Issue by insanity54** [insanity54/futureporn#56]
futureporn-scout does a really good job at detecting chaturbate links and generating a markdown file which serves as a datum for futureporn... but the problem is that usually the text content with the CB link is usually a followup to the main text content which belongs to a parent tweet.

Example tweet https://twitter.com/ProjektMelody/status/1555694026376060966 has text content, "i'm live on a sussy but cool site here ----->" which is meant to direct people to CB, but lacks the context or theme of the stream as indicated in the parent tweet, "Sup Race Fans!!!!! Ready to get your gaming on??"

TL;DR: The parent tweet contains the high quality, themed text content that I want the announceTitle to contain, but it's a descendant tweet that is getting used instead.
